core/templates/dev/head/components/ruleEditor.js

```diff
@@ -227,6 +229,9 @@ oppia.directive('ruleEditor', ['$log', function($log) {
               'isWeighted': false,
               'isLabeled': false
             };
+          } else if(_answerChoices){
```

Comment:
nit: add space before '(' and add space after ')'. Generally, follow the [Google JS style guide](http://google-styleguide.googlecode.com/svn/trunk/javascriptguide.xml#Code_formatting).